ts/src/sdk/nodejs/createProviderSDK.ts

```diff
@@ -0,0 +1,33 @@
+import { createSDK } from "../../generated/createProviderSDK.ts";
+import { createGrpcTransport } from "../../transport/grpc/createGrpcTransport.ts";
+import type { PickByPath } from "../../utils/types.ts";
+
+type ProviderSDK = PickByPath<ReturnType<typeof createSDK>, "akash.provider.v1">;
+
+export function createProviderSDK(options: ProviderSDKOptions): ProviderSDK {
+  const certificateOptions = options.authentication?.type === "certificate"
+    ? {
+        cert: options.authentication?.cert,
+        key: options.authentication?.key,
+      }
+    : null;
```

Comment:
question(non-blocking): I assume this would be jwt and we need to add the implementation to chain-sdk eventually?

Reply:
@baktun14 some of the provider details do not require authentication, so technically authentication is optional. 

But at the end - yes. We will need to add JWT support here as well. I will think about it when I have possibility to test both auth mechanisms on real servers